aiohttp/web_runner.py

```diff
@@ -74,30 +71,11 @@ async def start(self) -> None:
 
     async def stop(self) -> None:
         self._runner._check_site(self)
-        if self._server is None:
-            self._runner._unreg_site(self)
-            return  # not started yet
-        self._server.close()
-        # named pipes do not have wait_closed property
-        if hasattr(self._server, "wait_closed"):
-            await self._server.wait_closed()
```

Comment:
So, this method in Python 3.12 actually waits until all connections are closed. But, we haven't even started to shutdown our connections yet, so this is clearly wrong. I don't think we actually need to call it at all because the runner basically waits on the connections after cancelling them.

On older versions of Python, this method does literally nothing, so this code only worked in the past by accident.